framework/build.xml

```diff
@@ -226,6 +226,11 @@
             <package name="org.checkerframework.common.*"/>
             <package name="org.checkerframework.checker.*"/>
 
+            <fileset dir="${checker.loc}/${src}">
```

Comment:
Does the javadoc target here not need the same modifications as in checker/build.xml?